setup.py

```diff
@@ -80,15 +80,15 @@ def run_tests(self):
     keywords="api graphql protocol rest relay graphene",
     packages=find_packages(exclude=["tests", "tests.*", "examples"]),
     install_requires=[
-        "graphql-core>=2.1,<3",
-        "graphql-relay>=0.4.5,<1",
-        "aniso8601>=3,<=6.0.*",
+        "graphql-core>=3.0.0a0,<4",
+        "graphql-relay>=3.0.0a0,<4",
+        "aniso8601~=6.0.0",
     ],
     tests_require=tests_require,
     extras_require={
         "test": tests_require,
-        "django": ["graphene-django"],
-        "sqlalchemy": ["graphene-sqlalchemy"],
+        # "django": ["graphene-django"],
+        # "sqlalchemy": ["graphene-sqlalchemy"],
```

Comment:
Should these be removed?

Reply:
I think so. It's much better and simpler to install graphene-django/sqlalchemy which then require a matching version of graphene. Also, it's unclear why graphene-django gets that extra treatment while graphene-flask does not. Better to keep graphene framework angostic.